tests/foreman/longrun/test_capsule_loadbalancer.py

```diff
@@ -16,71 +16,206 @@
 
 :Upstream: No
 """
+import queue
+import threading
+from time import sleep
+
 import pytest
+from broker import VMBroker
+from nailgun import entities
+
+from robottelo.api.utils import promote
+from robottelo.cli.capsule import Capsule as ContentCapsule
+from robottelo.config import settings
+from robottelo.helpers import get_data_file
+from robottelo.helpers import InstallerCommand
+from robottelo.hosts import Capsule
+from robottelo.hosts import ContentHost
+
+
+@pytest.fixture(scope='session')
+def get_hosts_from_broker():
```

Comment:
I want to come back to this next week.

Reply:
Sure!

Reply:
Broker's ability to multiprocess checkouts is based on the hosts using the same arguments.